pycbc/waveform/ringdown.py

```diff
@@ -259,21 +259,37 @@ def lm_deltaf(damping_times, modes):
 
     return min(df.values())
 
-# Functions for tapering #######################################################
+# Spherical harmonics #########################################################
 
-def apply_taper(delta_t, taper, f_0, tau, amp, phi):
+def spher_harms(l, m, inclination):
+    """Return spherical harmonic polarizations
+    """
+
+    # FIXME: we are using spin -2 weighted spherical harmonics for now,
+    # when possible switch to spheroidal harmonics.
+    Y_lm = lal.SpinWeightedSphericalHarmonic(inclination, 0., -2, l, m).real
+    Y_lminusm = lal.SpinWeightedSphericalHarmonic(inclination, 0., -2, l, -m).real
+    Y_plus = Y_lm + (-1)**l * Y_lminusm
+    Y_cross = Y_lm - (-1)**l * Y_lminusm
+
+    return Y_plus, Y_cross
+
+# Functions for tapering ######################################################
+
+def apply_taper(delta_t, taper, f_0, tau, amp, phi, l, m, inclination):
     """Return tapering window.
     """
 
     # Times of tapering do not include t=0
     taper_times = -numpy.arange(delta_t, taper*tau, delta_t)
     taper_times.sort()
```

Comment:
Do this instead:
```
taper_times = -numpy.arange(1, int(taper*tau/delta_t))[::-1] * delta_t
```
That'll ensure that it starts at exactly -delat_t. Also, the `[::-1]` will invert the order, which negates needing the sort on the next line.